io.github.ja2_stracciatella.JA2-Stracciatella.appdata.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>io.github.ja2_stracciatella.JA2-Stracciatella</id>
+    <launchable type="desktop-id">io.github.ja2_stracciatella.JA2-Stracciatella.desktop</launchable>
+    <name>Jagged Alliance 2 Stracciatella</name>
+    <summary>An improved, cross-platform, stable Jagged Alliance 2 runtime</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-proprietary=https://github.com/ja2-stracciatella/ja2-stracciatella/blob/master/SFI%20Source%20Code%20license%20agreement.txt</project_license>
```

Comment:
Add developer_name

Reply:
No appstream-glib does not support developer, even though developer_name is deprecated. You can switch to it when Flathub switches to libappstream.

Reply:
Oh well…  I added `developer_name` then.